extensions/qotp/README.md

```diff
@@ -0,0 +1,9 @@
+# QOTP
+
+## pin 获取
+
+qotp.corp 平台,用户输入的六位密码
+
+## [secret 获取教程](https://sourl.cn/4NfR9x)
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English. Therefore, please avoid introducing your custom way to localize your extension.